easybuild/easyconfigs/l/libctl/libctl-4.0.0-intel-2020a.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'libctl'
+version = '4.0.0'
+
+homepage = 'https://github.com/stevengj/libctl'
+description = """libctl is a free Guile-based library implementing flexible control files for scientific simulations."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'optarch': True, 'pic': True}
```

No need to enable `optarch`, that's the default

```suggestion
toolchainopts = {'pic': True}
```